lib/index.js

```diff
@@ -4,6 +4,10 @@ var stringify = require('./stringify');
 var parse = require('./parse');
 
 module.exports = {
+    formats: {
```

Comment:
could this be stored in a separate file, and required in here? that way `stringify.js` could require it, and use the constant rather than the string